repository-data/development/src/main/resources/hcm-content/content/documents/administration/lks/lksconfiguration.yaml

```diff
@@ -0,0 +1,52 @@
+/content/documents/administration/lks/lksconfiguration:
```

Comment:
I don't think it is good idea to put the configuration as resource bundle. That is out of the main purpose of resource bundle. Should we find other way ?

Reply:
Sure, please let me know what you have in mind.

Reply:
For this case of storing the blogsOverview's URL, I am thinking
- First we created hstSiteMapItem for the blogsOverview and set its `hst:refId` to a fixed value like `blogs`
- Create the HstLink for blogsOverview page using the fixed refId
```
final HstLink hstLink = request.getRequestContext().getHstLinkCreator().createByRefId("blogs", request.getRequestContext().getResolvedSiteMapItem().getResolvedMount().getMount());
```
- From the HstLink we can have the URL somehow.

Reply:
@thojava - Updated `uk.nhs.hee.web.components.BlogPostComponent` to generate Blog Overview URL for categories based on `blogs` SiteMapItem RefId and defaults to `/blogs` URL if `blogs` SiteMapItem RefId isn't available in the repository yet. Note that the creation of SiteMapItem with RefId `blogs` should be done as part of blogs overview page build ticket.